cmd/snap-confine/mount-support.h

```diff
@@ -66,4 +66,13 @@ void sc_ensure_shared_snap_mount(void);
 void sc_setup_user_mounts(struct sc_apparmor *apparmor, int snap_update_ns_fd,
 			  const char *snap_name);
 
+/**
+ * Set up mappings for snap instance.
+ *
+ * Perform setup of snap directory mappings of the snap being run is a parallel
+ * installed one. This ensures that snap directories appear the same for snaps
+ * with instance key and the ones without.
+ */
```

Comment:
I would love to see a pair of functions (or an argument) that identifiers if this is the global (classic), per-snap (strict,devmode) or per-snap and per-user mount namespace we are expected to modify. As it stands now I think this part is incorrect.